app/assess/models/fund.py

```diff
@@ -1,16 +1,15 @@
 from dataclasses import dataclass
 from typing import List
 
+from app.assess.models.round import Round
 from slugify import slugify
 
-from .application import Application
-
 
 @dataclass
 class Fund:
     name: str
     _identifier: str = None
-    applications: List[Application] = None
+    rounds: List[Round] = None
 
     @property
     def identifier(self):
```

Comment:
do we still need @property still?

Reply:
I agree, you're right, every fund should have an id already here. Removing.